Makefile

```diff
@@ -0,0 +1,28 @@
+# Current version of the project.
```

Comment:
This Makefile isn't invoked by our CI/CD test infrastructure so the changes wouldn't actually be picked up by the released artifacts.

The current approach is to use Python scripts to invoke the build steps (bazel has been discussed as a possible solution).

This is the Python function that invokes go build in our continuous tests
https://github.com/tensorflow/k8s/blob/master/py/release.py#L138

Our continuous release process is just a cron job that invokes that script
https://github.com/tensorflow/k8s/blob/master/release/releaser.yaml

So I think you just want to update the go command line here
https://github.com/tensorflow/k8s/blob/master/py/release.py#L138

And then the changes would be picked up.